scripts/judge/update-judge-isSeniorJudge.ts

```diff
@@ -1,26 +1,76 @@
+import { MAX_ELASTICSEARCH_PAGINATION } from '@shared/business/entities/EntityConstants';
 import { User } from '@shared/business/entities/User';
-import { requireEnvVars } from '../../shared/admin-tools/util';
-requireEnvVars(['ENV', 'REGION', 'DYNAMODB_TABLE_NAME', 'DYNAMODB_ENDPOINT']);
 import { createApplicationContext } from '@web-api/applicationContext';
+import { requireEnvVars } from '../../shared/admin-tools/util';
+import { search } from '@web-api/persistence/elasticsearch/searchClient';
 
-/**
-How to Run:
-
-npx ts-node --transpile-only scripts/judgeUpdates/update-judge-isSeniorJudge.ts
-*/
+requireEnvVars([
+  'DYNAMODB_ENDPOINT',
+  'DYNAMODB_TABLE_NAME',
+  'ELASTICSEARCH_ENDPOINT',
+  'ENV',
+  'REGION',
+]);
 
-// ******************************** INPUTS ******************************
-const judgesToUpdateIds: { userId: string; isSeniorJudge: boolean }[] = [
-  {
-    isSeniorJudge: true,
-    userId: '111111-11111-1111-111111-111111',
-  },
+const seniorJudges = [
```

Comment:
question for curiosity: how often does this list change?

Reply:
probably often enough that we'll need a comment warning that it changes